README-sandbox.md

```diff
@@ -127,7 +133,8 @@ environment variable, as it might be required in some setups:
 $ export MINA_ROSETTA_MAX_DB_POOL_SIZE=128
 ```
 
-Assuming `mina` CLI is in you path, run the following command:
+Assuming `mina` CLI is in you path, run the following command 
+from the mina directory and ensure the config-directory is an absolute-path:
```

Comment:
```suggestion
Assuming `mina` CLI is in you path, run the following command, 
making sure that required environment variables are properly set
(by Nix shell or otherwise):
```

The information about the absolute path I'd move below to where
`--config-directory` option is explained. The issue does not always
appear, so I'd append: "In case the daemon fails complaining about
write access to this directory, try providing an absolute path
rather than relative one", or something like that to the paragraph.